README.md

```diff
@@ -1 +1,64 @@
-# leader_board
\ No newline at end of file
+# Leader Board
+
+## Description
+
+> The main purposes of the project are:
+
+- Understand how to use `Gitflow`
+- Understand how to use `Webpack`
+- Create markup file and add markup for leaderboard
+- Style them using CSS
+- How to use `import` and `export`
+
+## Live View
+
+[`https://ridaarif98.github.io/leader_board/dist/`](https://ridaarif98.github.io/leader_board/dist/)
+
+## Built With
+
+This project is build with:
+
+- HTML
+- CSS
+- Javascript
+- Webpack
+
+
+## Getting Started
+
+# To get a local copy of this repository kindly follow the steps below:
+
+- In your local PC, open your terminal or command prompt in the folder you would like to clone this repository into
+- Make sure you have Node NPM installed on your computer
+-  Clone or download this repo on your machine:` git clone https://github.com/ridaarif98/leader_board.git`
+- Enter project directory with command `cd leader_board`
+
+# Install 
+
+- Run `npm install`
+- Run `run build` 
+
+# Usage 
+
+- Run `npm start`
```

Comment:
## [OPTIONAL]
* I suggest you remove the command `npm start` as you don't currently have that script in your package.json file 🙏